src/IssueViz.Security.UnitTests/IssueVizDisplayPositionCalculatorTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.Text;
+using Moq;
+using SonarLint.VisualStudio.Core.Analysis;
+using SonarLint.VisualStudio.IssueVisualization.Models;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Security.UnitTests
+{
+    [TestClass]
+    public class IssueVizDisplayPositionCalculatorTests
+    {
+        [TestMethod]
+        [DataRow(true)]
+        [DataRow(false)]
+        public void Line_NoSpan_ReturnsHotspotStartLine(bool spanIsNull)
+        {
+            const int originalLineNumber = 123;
+            var issueViz = CreateIssueVizWithoutSpan(spanIsNull, originalLineNumber: originalLineNumber);
+
+            var testSubject = CreateTestSubject();
+            testSubject.GetLine(issueViz).Should().Be(originalLineNumber);
```

Comment:
Cut and pasted from the view model tests with a few changes... but I'm not going to tell you what they are - MWAA-HA-HA-HA-HA!!!! ;-)

Ok, 'CreateTestSubject' is obviously different, and the calls to the `Line` and `Column` properties are now calls to `GetLine` and `GetProperty`. Everything else is identical.

Reply:
touché!